installing/README.md

```diff
@@ -0,0 +1,10 @@
+# Installing Nix
```

Comment:
I don't think that's a good idea if we want to follow the rustbyexample structure. It's a "how" book, not a "what and why". Maybe some closing remarks at the end?